frontend/src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,53 @@
+.NavbarItems {
+  background: linear-gradient(90deg, #957DAD 0%, #e0bbe4 100%);
```

Comment:
Let's stick to convention for styling:

Component name Navbar === root class name Navbar

Working with logo in Navbar === Navbar-Logo
Working with menu === Navbar-Menu